getting-started/GettingStarted.md

```diff
@@ -0,0 +1,48 @@
+# Getting Started: Publishing a Kubernetes App on Azure Marketplace
+
+### Introduction
+Thank you for joining us in our Kubernetes Apps public preview program. We’re delighted to have you participate in our vibrant and ever-growing ecosystem, where you can publish and monetize your Kubernetes application. This document has all the necessary steps to get you started and can enable you to publish your application in 1-2 weeks to the Kubernetes marketplace in Azure.
+
+### Pre-requisites
+You must meet the following criteria to be able to participate in the public preview:
+* The Kubernetes application should have a Helm-chart based application that is free of malware. We highly recommend doing a [pre-scan and fixing the vulnerabilities](https://github.com/bobmital/kubernetes-offer-samples/blob/05611a5c6485db980fe27fb8999af728382b547d/getting-started/Vulnerability-Scan.md) before you push the application through the pipeline.
```

Comment:
The link here is pointing to Bob's repo https://github.com/bobmital/kubernetes-offer-samples. Can you try specifying the relative link?